guides/common/modules/proc_upgrading-project-or-proxy-in-place-using-leapp.adoc

```diff
@@ -0,0 +1,228 @@
+[id="upgrading-{project-context}-or-proxy-in-place-using-leapp_{context}"]
+= Upgrading {Project} or {SmartProxy} to {EL} 9 in-place by using Leapp
+
+{Project} and {SmartProxy} are supported on both {EL} 8 and {EL} 9.
+You can upgrade your {Project} and {SmartProxy} operating system from {EL} 8 to {EL} 9. 
+When upgrading your {Project} or {SmartProxy} installation from {EL} 8 to {EL} 9, use the Leapp tool to upgrade as well as to help detect and resolve issues that could prevent you from upgrading successfully.
+
+.Prerequisites
+ifdef::satellite[]
+* Review Known Issues before you begin an upgrade.
+For more information, see {ReleaseNotesDocURL}ref_known-issues_assembly_introducing-red-hat-satellite[Known Issues in {ProjectName} {ProjectVersion}].
+endif::[]
+ifdef::foreman-el,katello[]
+* Review Known Issues before you begin an upgrade.
+For more information, see {BaseURL}Release_Notes/index-katello.html#katello-upgrade-warnings[Upgrade Warnings].
```

Comment:
```suggestion
For more information, see {ReleaseNotesBaseURL}[Release notes].
```
You can use this attribute as it's [defined for all builds](https://github.com/theforeman/foreman-documentation/blob/b9e577fb14e7b9f605f24d86dfdddc4d225d4b1f/guides/common/attributes-base.adoc#L18).
Also, because the Upgrade warnings ID depends on the build and isn't consistent with the build, let's just link to RNs as such.